### PR TITLE
User details on registration

### PR DESCRIPTION
**Pull request by KhadijaMahanga**
## Description

- Include Default From Email setting
- Update custom signup, to save `first_name` and `last_name`

Fixes # (issue)

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by kilemensi — COMMENTED** [review]
👍🏽 

----

Why not override: [`get_cleaned_data`](https://github.com/Tivix/django-rest-auth/blob/3c36004c44855cd25b535fe822b16d9e3fa1a3d1/rest_auth/registration/serializers.py#L199-L204)

This way, there is only one call to [save](https://github.com/Tivix/django-rest-auth/blob/3c36004c44855cd25b535fe822b16d9e3fa1a3d1/rest_auth/registration/serializers.py#L210) user data.

**Comment by KhadijaMahanga** [comment]
@kilemensi yes, thats possible too

File: twoopstracker/settings.py [hunk 1]
```diff
@@ -229,7 +229,15 @@
...
 )
 
+# Custom Register Serializer
+# https://dj-rest-auth.readthedocs.io/en/latest/configuration.html
+REST_AUTH_REGISTER_SERIALIZERS = {
+    "REGISTER_SERIALIZER": "twoopstracker.authentication.serializers.CustomRegisterSerializer",
+}
 
+DEFAULT_FROM_EMAIL = env.str(
+    "DEFAULT_FROM_EMAIL", "no-reply@trolltracker.investigate.africa"
```
Comment (kilemensi) [inline twoopstracker/settings.py:239]:
```suggestion
    "DEFAULT_FROM_EMAIL", "webmaster@localhost"
```

... after all, the project is `TwoopsTracker`, where does `TrollTracker` even come from 😎

**Review by kilemensi — APPROVED** [review]
🚀

**Pull request merged**